addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -23,78 +23,78 @@
                                 <t t-set="rating_count" t-value="record.rating_count"/>
                             </t>
                         </div>
-                    </div>
-                    <div class="o_carousel_product_card_footer card-footer d-flex align-items-center">
-                        <div>
-                            <t t-call="website_sale.price_dynamic_filter_template_product_product"/>
+                        <div class="w-100 d-flex flex-wrap flex-md-column flex-lg-row align-items-center align-self-end justify-content-between mt-3">
```

Comment:
Just making sure: has this been retested since BS5?

Reply:
I did adapt it to BS5. Though I wasn't the one who made those templates, maybe it's worth if @Brieuc-brd can have a final look?

Reply:
I would not mind having his opinion but we are a bit too rushed to delay this further (I am already not a fan a merging it now but since some people really wants it...). Let's not delay the merge (I'll merge tonight or tomorrow max), I was merely making sure you had a look at potential BS5 problems :+1: